firefox_desktop/views/urlbar_events.view.lkml

```diff
@@ -1,223 +1,331 @@
+    description: "The type of annoyance: help, inaccurate_location, not_interested, not_relevant, show_less_frequently"
+  }
 
-dimension: engaged_result_type {
-  hidden: yes
-}
+  dimension: engaged_result_type {
+    hidden: yes
+  }
 
-dimension: event_name {
-  hidden:  yes
-}
+  dimension: engagement_type {
+    hidden: yes
+  }
 
-dimension: event_timestamp {
-  hidden:  yes
-}
+  dimension: event_action {
+    group_label: "Urlbar-specific filters"
+    description: "The type of urlbar event: engagement, abandonment or annoyance."
+  }
 
-dimension: experiments {
-  hidden:  yes
-}
+  dimension: session_action_type {
+    group_label: "Urlbar-specific filters"
+    description: "Don't use this field. It's the same as `Event Action`. It is included only for backwards compatibility on some older dashboards."
+  }
 
-dimension: first_result_type {
-  hidden:  yes
-}
+  dimension: event_id {
+    hidden: yes
+  }
 
-dimension: glean_client_id {
-  hidden:  yes
-}
+  dimension: event_name {
+    hidden:  yes
+  }
 
-dimension: is_terminal {
-  hidden:  yes
-}
+  dimension: event_timestamp {
+    hidden:  yes
+  }
 
-dimension: legacy_telemetry_client_id {
-  hidden:  yes
-}
+  dimension: experiments {
+    hidden:  yes
+  }
 
-dimension: normalized_channel {
-  sql: ${TABLE}.normalized_channel ;;
-  type: string
-}
+  dimension: glean_client_id {
+    hidden:  yes
+  }
 
-dimension: normalized_country_code {
-  sql: ${TABLE}.normalized_country_code ;;
-  type: string
-}
+  dimension: interaction {
+    hidden:  yes
+  }
 
-dimension: num_add_on_impressions {
-  hidden:  yes
-}
+  dimension: is_terminal {
+    hidden:  yes
+  }
 
-dimension: num_admarketplace_sponsored_impressions {
-  hidden:  yes
-}
+  dimension: legacy_telemetry_client_id {
+    hidden:  yes
+  }
 
-dimension: num_bookmark_impressions {
-  hidden:  yes
-}
+  dimension: normalized_channel {
+    sql: ${TABLE}.normalized_channel ;;
+    type: string
+  }
 
-dimension: num_chars_typed {
-  hidden:  yes
-}
+  dimension: normalized_country_code {
+    sql: ${TABLE}.normalized_country_code ;;
+    type: string
+  }
+
+  dimension: normalized_engine {
+    sql: ${TABLE}.normalized_engine ;;
+    type: string
+  }
 
-measure: average_characters_typed {
+  dimension: num_chars_typed {
+    hidden:  yes
+  }
+
+  measure: average_characters_typed {
     type: number
     sql: AVG(${TABLE}.num_chars_typed) ;;
   }
 
-dimension: num_default_partner_search_suggestion_impressions {
-  hidden:  yes
-}
+  dimension: num_total_results {
+    hidden:  yes
+  }
 
-dimension: num_history_impressions {
-  hidden:  yes
-}
+  dimension: pref_data_collection {
+    hidden: yes
+  }
 
-dimension: num_navigational_impressions {
-  hidden:  yes
-}
+  dimension: data_collection_enabled {
+    sql: ${TABLE}.pref_data_collection ;;
+    type: yesno
+    group_label: "User Preferences"
+    description: "Whether or not the checkbox for sharing search terms data is checked in about:preferences"
```

Comment:
Would it be worthwhile using the term "suggest online" here?

Reply:
yep, updated